File: backend/src/main/java/br/com/importaai/domain/model/Moeda.java
package br.com.importaai.domain.model;

public enum Moeda {
    BRL("R$"),
    USD("$"),
    EUR("€"),
    CNY("¥");

    private final String simbolo;

    Moeda(String simbolo) {
        this.simbolo = simbolo;
    }

    public String getSimbolo() {
        return simbolo;
    }
}

File: frontend/src/types/moeda.ts
export type TipoMoeda = 
    | 'BRL' 
    | 'USD' 
    | 'CNY'
    | 'EUR'

export interface Moeda {
    nome: string
    pais: string
    sigla: string
    tipo: TipoMoeda
    valor: number
    atualizacao: string
    manual?: boolean
}